lib/instrumentation/index.js

```diff
@@ -356,6 +356,10 @@ Instrumentation.prototype.addEndedSpan = function (span) {
     return
   }
 
+  if (!span.isRecorded()) {
+    return
```

Comment:
```suggestion
    span.transaction.captureDroppedSpan(span)
    return
```

Otherwise dropped span stats won't ever be collected.

Reply:
ha -- guess I didn't hit ⌘s before committing that one.